apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -87,10 +87,13 @@ public function get_nux_status() {
 
 		$show_welcome_guide = $this->show_wpcom_welcome_guide( $nux_status );
 
+		$should_open_patterns_panel = get_option( 'current_theme' ) === 'Blank Canvas';
```

Comment:
Somehow using `get_option( 'is_blank_canvas' )` doesn't work, the property doesn't exist, although the site was created via `/new?flags=gutenboarding/alpha-templates`. For now I'm checking via the template name. Maybe something to investigate?

Reply:
I believe that's because D60468-code hasn't been merged yet.

You'd need to:

1. Apply D60468-code to your sandbox
2. Create a site that uses the Blank Canvas design
3. Add the newly created site to your sandbox
4. At that point, `get_option( 'is_blank_canvas' )` _should_ work

Reply:
Oh I see the status is **Accepted** not **Merged**!  That's why!

Reply:
Updated that diff to use the site option we're now passing when creating the site. See https://github.com/Automattic/wp-calypso/pull/52288/commits/b6b2d670bb7de42499fe437bb9b6f45a514ffd33